### ISSUE TITLE
inconsistent cdn.edge.is_cacheable value logged

### ISSUE THREAD

**Issue by stefan-guggisberg** [adobe/helix-logging#344]
**Description**
Log output:
```
       "cache_status"  :  "HIT" ,
       "is_cacheable"  :  false ,
```

**To Reproduce**
Look at any Fastly log in Coralogix and check the `cdn.edge.is_cacheable` and `cdn.edge.cache_status` values.

**Expected behavior**
Accurate `cdn.edge.is_cacheable` value, e.g. `true` if `cdn.edge.cache_status` was `HIT`.

Relevant code: https://github.com/adobe/helix-logging/blob/master/src/coralogix/schema.js#L111-L114

**Comment by stefan-guggisberg** [comment]
@trieloff What's the idea here? Why a regex and not simply log `fastly_info.state`?
```
        cache_status: str(
          vcl`regsub(fastly_info.state, "^(HIT-(SYNTH)|(HITPASS|HIT|MISS|PASS|ERROR|PIPE)).*", "\\\\2\\\\3")`,
        ),
```

BTW: The regex doesn't seem to cover all the possible states as documented [here](https://support.fastly.com/hc/en-us/community/posts/360040168391/comments/360004718351).

**Comment by trieloff** [comment]
I copied it from some Fastly documentation, but don't remember the details. Maybe we should just log info_state.

**Comment by stefan-guggisberg** [comment]
> Maybe we should just log info_state.
+1

What about `is_cacheable`? It's redundant (and incorrect). Ok to drop it?

**Comment by trieloff** [comment]
Has this been fixed in #345?

**Comment by stefan-guggisberg** [comment]
yup, closing
